subgraph/subgraph.yaml

```diff
@@ -1,33 +1,35 @@
-specVersion: 0.0.5
 schema:
   file: ./schema.graphql
 dataSources:
   - kind: ethereum
     name: FleekNFA
     network: mumbai
     source:
-      address: "0x11cd77eac1a0883de9435b5586474903a2a084cd" # <- Proxy Contract | Current implementation contract: 0x113169680A417e4dB4AD11832486FF317fB4CB36
+      address: "0x550Ee47Fa9E0B81c1b9C394FeE62Fe699a955519" # <- Proxy Contract | Current implementation contract: 0x9e4a318c788e0097a24146fe7041a0cd93939d56
       abi: FleekNFA
-      startBlock: 32071579
+      startBlock: 32373064
     mapping:
       kind: ethereum/events
       apiVersion: 0.0.7
       language: wasm/assemblyscript
       entities:
         - Approval
         - ApprovalForAll
-        - CollectionRoleGranted
-        - CollectionRoleRevoked
         - MetadataUpdate
         - NewMint
-        - TokenRoleGranted
-        - TokenRoleRevoked
         - Transfer
         - Token
         - Owner
         - Controller
         - CollectionOwner
         - GithubRepository
+        - AccessPoint
+        - NewAccessPoint
```

Comment:
We didn't continue the discussion on storing event entities and postponed it IIRC.

Reply:
YeahbBut I mean I don't see that entity defined in `schema.graphql`.

Reply:
that's no problem

Reply:
I'm good either way too but since we didn't implement the NewAccessPoint entitty yet, and we didnt decide on it like you said, should we remove this line to keep it clean and consistent?

Reply:
I don't get why we would partially put something that we are undecided on and it's especially not clean cuz it doesn't map to an entity we defined. So yeah I think we should remove this line until we decide to implement the NewAccessPoint entity becaues its not evne in schema.graphql yet.